src/sssom/parsers.py

```diff
@@ -68,6 +68,8 @@
 
 logging = _logging.getLogger(__name__)
 
+DBXREF_URI = "http://www.geneontology.org/formats/oboInOwl#hasDbXref"
```

Comment:
This should be imported from constants.py (probably its already there)

Reply:
Done in 6c09ed3